File: findMedianSortedArrays.ts
function findMedianSortedArrays(nums1: number[], nums2: number[]): number {
    const sortedArray = [...nums1, ...nums2].sort((n1, n2) => n1 - n2);
    const arrayLength = sortedArray.length;
    if (arrayLength % 2 === 0) {
        return (sortedArray[arrayLength / 2 - 1] + sortedArray[arrayLength / 2]) / 2;
    }
    return sortedArray[Math.floor(arrayLength / 2)];
};

File: find_median_sorted_arrays.py
from typing import List


class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        nums = nums1 + nums2
        nums.sort()
        num_len = len(nums)
        median = num_len // 2
        if num_len % 2 == 0:
            return (nums[median - 1] + nums[median]) / 2
        return nums[median]
